compair/api/users.py

```diff
@@ -397,6 +403,24 @@ def get(self):
                 Course.sandbox == params['includeSandbox']
             )
 
+        if params['period'] != None:
+            now = dateutil.parser.parse(datetime.datetime.utcnow().replace(tzinfo=pytz.utc).isoformat())
+            if params['period'] == 'upcoming':
+                query = query.filter(
+                    Course.start_date > now
+                )
+            elif params['period'] == 'active':
+                query = query.filter(or_(
```

Comment:
Good handling all the situations but I think it can be simplified to 
```
 query = query.filter(and_(
    or_(Course.start_date == None, Course.start_date <= now),
    or_(Course.end_date == None, Course.end_date >= now),
))
```

Reply:
Cool, thanks!